ckan/Dockerfile.dev

```diff
@@ -44,3 +44,24 @@ RUN for d in $APP_DIR/patches/*; do \
             done ; \
         fi ; \
     done
+
+# Copy any files in setup to the APP_DIR
+COPY setup $APP_DIR
+
+# Setup pycsw
+WORKDIR $SRC_EXTENSIONS_DIR
+
+RUN git clone  --branch 2.4.0 https://github.com/geopython/pycsw.git ; \
```

Comment:
I think it's better to use `&&` instead of `;` between the statements because then it will stop executing if one of the commands fails.

Reply:
ok, will update